pkg/storaged/nfs-mounts.py

```diff
@@ -65,7 +66,8 @@ def index_tab(tab):
 
 
 def modify_tab(name, modify):
-    lines = open(name, "r").read().split("\n")
+    with open(name, "r") as f:
+        lines = f.read().split("\n")
```

Comment:
...and that one is (approximately)

`   lines = f.readlines()`

Since you strip the lines in the loop below, it ought to be equivalent.

Reply:
Done.